docker build fails on SB master

```
[ 99%] Performing install step for 'SIRF'
...
Install the project...
-- Install configuration: "Release"
-- Installing: /opt/SIRF-SuperBuild/INSTALL/include/sirf/common/version.h
CMake Error at cmake_install.cmake:61 (file):
  file cannot create directory: /sirf.  Maybe need administrative privileges.


Makefile:138: recipe for target 'install' failed
make[3]: *** [install] Error 1
CMakeFiles/SIRF.dir/build.make:93: recipe for target 'builds/SIRF/stamp/SIRF-install' failed
make[2]: *** [builds/SIRF/stamp/SIRF-install] Error 2
CMakeFiles/Makefile2:192: recipe for target 'CMakeFiles/SIRF.dir/all' failed
make[1]: *** [CMakeFiles/SIRF.dir/all] Error 2
Makefile:114: recipe for target 'all' failed
make: *** [all] Error 2
```

It suggests that it is trying to install in the wrong directory:

```
CMake Error at cmake_install.cmake:61 (file):
  file cannot create directory: /sirf.  Maybe need administrative privileges.
```

`cmake_install.cmake` at line 61 has indeed 
```cmake
file(INSTALL DESTINATION "/sirf" TYPE FILE FILES "/opt/SIRF-SuperBuild/builds/SIRF/build/cmake/config.py")
```

However the CMake cache seems to be configured correctly

```
CMAKE_INSTALL_PREFIX             /opt/SIRF-SuperBuild/INSTALL
 CMAKE_LIBRARY_PATH               /opt/SIRF-SuperBuild/INSTALL/lib
 CMAKE_PREFIX_PATH                /opt/SIRF-SuperBuild/INSTALL
```

--- a/CHANGES.md
+++ b/CHANGES.md
@@ -1,6 +1,7 @@
 # ChangeLog
 ## v3.2.0
 
+- Drop Python 2 support
 - CMake minimum required version is set to 3.16.2, as it supports Boost 1.72
 - VirtualBox VM is created from the SIRF-SuperBuild repository
 - Docker images install CIL via conda, so setting `BUILD_CIL=OFF` in the docker build

--- a/SuperBuild.cmake
+++ b/SuperBuild.cmake
@@ -95,13 +95,9 @@ if (DISABLE_PYTHON)
   message(STATUS "Python support disabled")
 else()
 
-  # Attempt to make Python settings consistent
-  set(PYVER 0 CACHE STRING "Python version")
-  if(PYVER EQUAL 0)
-    find_package(PythonInterp)
-  else()
-    find_package(PythonInterp ${PYVER})
-  endif()
+  # only Python 3 is supported
+  find_package(PythonInterp 3)
+  
   if (PYTHONINTERP_FOUND)
     set(Python_ADDITIONAL_VERSIONS ${PYTHON_VERSION_STRING})
     message(STATUS "Found PYTHON_EXECUTABLE=${PYTHON_EXECUTABLE}")

